packages/@aws-cdk/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -16,6 +17,18 @@ export type AwsSdkMetadata = {[key: string]: any};
 
 const awsSdkMetadata: AwsSdkMetadata = metadata;
 
+/**
+ * Reference to the physical resource id that can be passed to the AWS operation as a parameter.
+ */
+export class PhysicalResourceIdReference {
```

Comment:
Can you check and error if this value is used in the `onCreate` parameter block?